console/src/components/Services/Data/TableBrowseRows/ViewRows.js

```diff
@@ -759,9 +763,7 @@ const ViewRows = ({
           <b className={styles.padd_small_right}>Selected:</b>
           {selectedRows.length}
           <button
-            className={`${styles.add_mar_right_small} btn btn-xs btn-default ${
-              styles.bulkDeleteButton
-            }`}
+            className={`${styles.add_mar_right_small} btn btn-xs btn-default ${styles.bulkDeleteButton}`}
```

Comment:
This was done by the linter, not me.

Reply:
Yup, this is proper formatting. It was wrongly formatted on the master.